doc/game.md

```diff
@@ -150,6 +150,19 @@ built upon two methods:
 
 The `GameLoop` is used by all of Flame's `Game` implementations.
 
+# Pause/Resuming game execution
+
+A Flame `Game` can be paused and resumed in two ways:
+
+ - With the use of the `pauseEngine` and `resumeEngine` methods.
+ - By changing the `paused` attribute.
+
+When pausing a Flame `Game`, the `GameLoop` is effectively paused, meaning that no updates and new
+renders will happen until it is resumed.
+
+To pause/resume the update of custom components, custom logic needs to be implemented on each
+component.
+
```

Comment:
Line 195 in this file is btw wrong :-) it gets a context and a game variable

Reply:
Empty line
```suggestion
```